### PR TITLE
Update documentation on public key format and re-add tests

### PR DESCRIPTION
**Pull request by alexcope**
The `public_key_id` SHOULD be an RFC3986 conformant URI. We can not enforce any public key format without breaking backwards comparability. 

MUST and SHOULD are being used as defined in [rfc2119](https://www.ietf.org/rfc/rfc2119.txt).

### CODE REVIEW

**Comment by aysylu** [comment]
Could we please also address the revert to the fixes to tests as suggested by @steakunderscore in https://github.com/grafeas/grafeas/pull/444#issuecomment-629505126?

File: proto/v1/common.proto [hunk 1]
```diff
@@ -84,7 +84,7 @@ message Signature {
 
   // The identifier for the public key that verifies this signature.
   //   * The `public_key_id` is required.
-  //   * The `public_key_id` MUST be an RFC3986 conformant URI.
+  //   * The `public_key_id` SHOULD be an RFC3986 conformant URI.
```
Comment (aysylu) [inline proto/v1/common.proto:87]: Let's also make the same update to the v1beta1/common.proto

**Comment by steakunderscore** [comment]
I assume you'll need to regenerate the committed go code and swagger output:
```
proto/v1beta1/common_go_proto/common.pb.go
proto/v1beta1/swagger/grafeas.swagger.json
```

**Comment by alexcope** [comment]
> I assume you'll need to regenerate the committed go code and swagger output:
> 
> ```
> proto/v1beta1/common_go_proto/common.pb.go
> proto/v1beta1/swagger/grafeas.swagger.json
> ```

done, and re-added the tests from pr #432

File: proto/v1beta1/vulnerability_go_proto/vulnerability.pb.go [hunk 2]
```diff
@@ -379,8 +379,10 @@ type Vulnerability_WindowsDetail_KnowledgeBase struct {
 func (m *Vulnerability_WindowsDetail_KnowledgeBase) Reset() {
 	*m = Vulnerability_WindowsDetail_KnowledgeBase{}
 }
-func (m *Vulnerability_WindowsDetail_KnowledgeBase) String() string { return proto.CompactTextString(m) }
```
Comment (alexcope) [inline proto/v1beta1/vulnerability_go_proto/vulnerability.pb.go:382]: I'm not sure why this is formatting slightly differently for me. Should I manually revert the changes to this file?

Comment (aysylu) [reply proto/v1beta1/vulnerability_go_proto/vulnerability.pb.go:382]: This is strange, yes. Go formatting doesn't really restrict the number of chars per line. Let's leave this as is and come back if it starts causing issues, which I don't think it will. Thanks for flagging this!

Comment (aysylu) [reply proto/v1beta1/vulnerability_go_proto/vulnerability.pb.go:382]: Also, swagger sometimes just changes the code it generates or the formatting, so you may see a similar thing in the future. It's usually fine to let it do its thing.:)
